tests/event/event-image.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Wporg\Tests\Event;
+
+use GP_UnitTestCase;
+use Wporg\TranslationEvents\Routes\Event\Image_Route;
+use Wporg\TranslationEvents\Templates;
+use Wporg\TranslationEvents\Tests\Event_Factory;
+use Wporg\TranslationEvents\Urls;
+
+class Event_Image_Test extends GP_UnitTestCase {
+	/**
+	 * Test that when the header template is fired, it generates the social metadata.
+	 *
+	 * @return void
+	 */
+	public function test_gp_head_metadata() {
+		$html_title       = 'Test HTML Title';
+		$url              = Urls::events_home();
+		$html_description = 'This is a test description of the page.';
+		$image_url        = Urls::event_default_image();
+		$page_title       = 'Test Page Title';
+		Templates::header(
+			array(
+				'html_title'       => $html_title,
+				'url'              => $url,
+				'html_description' => $html_description,
+				'image_url'        => $image_url,
+				'page_title'       => $page_title,
+			),
+		);
+
+		ob_start();
+		do_action( 'gp_head' );
+		$output = ob_get_clean();
```

Comment:
Something is strange here. I noticed it because `Templates::header` outputs HTML already, so the phpunit output shows the HTML contents. But this also works, so do we even need to `do_action()` or are you testing something else here?

```suggestion
		ob_start();
		Templates::header(
			array(
				'html_title'       => $html_title,
				'url'              => $url,
				'html_description' => $html_description,
				'image_url'        => $image_url,
				'page_title'       => $page_title,
			),
		);
		$output = ob_get_clean();
```

Reply:
I used the `do_action` to fire the `add_action` in the template, but it is fired without this, so I removed it.